frontend/src/stores/filterState/FilterStateStore.ts

```diff
@@ -1,76 +1,97 @@
 /* istanbul ignore file */
 import { injectable } from 'inversify';
+import { Observable } from 'rxjs';
+import { map } from 'rxjs/operators';
 import SimpleStore from '../SimpleStore';
-import {
-  FilterLineState,
-  FilterPropertyState,
-  FilterState,
-} from './FilterState';
+import { FilterPropertyState, FilterState } from './FilterState';
 
 /**
  * Contains the current state of the filter.
  */
 @injectable()
 export default class FilterStateStore extends SimpleStore<FilterState> {
   protected getInitialValue(): FilterState {
-    return { edges: [], nodes: [] };
+    return new FilterState([], []);
+  }
+
+  public getValue(): FilterState {
+    const json = super.getValue();
```

Comment:
Why is this called json?

Reply:
Because the `super.getValue()` returns the raw object data like `{attr1: "smth", attr2: 42}` which has to be converted to a class type